src/other/broadcasting.jl

```diff
@@ -128,36 +128,41 @@ function Base.Broadcast.broadcast_unalias(dest::AbstractDataFrame, src)
     src
 end
 
+function _broadcast_unalias_helper(dest, scol, src, col2, wascopied)
```

Comment:
Make the signature more specific for clarity?

Reply:
ok. Regarding the `col1 == col2 && dcol === scol && continue` check it did not help that much, so I left it out.